package.json

```diff
@@ -20,7 +20,8 @@
     "test:watch": "jest --watch",
     "lint": "eslint . --ext .ts --max-warnings 0",
     "lint:fix": "npm run lint -- --fix",
-    "postinstall": "./scripts/postinstall.sh"
+    "postinstall": "./scripts/postinstall.sh",
```

Comment:
Hmmmmm I dont understand the diff ? i've changed this to use node script to fix #70 (https://github.com/Open-Attestation/open-attestation/blob/master/package.json#L23)

Reply:
(ok the MR is in conflict :D)

Reply:
lol fixed.